src/modules/checked.js

```diff
@@ -0,0 +1,23 @@
+import toDoItemsArray from './toDoItemsArray.js';
+
+const checkedBoxEls = document.getElementsByClassName('checkbox');
+
+const checked = () => {
+  const checkedBoxElsArray = [...checkedBoxEls];
+  checkedBoxElsArray.forEach((item) => {
+    item.addEventListener('click', () => {
+      const itmId = item.parentNode.parentNode.id;
+      const { completed } = toDoItemsArray[itmId];
+
+      if (completed) {
```

Comment:
This is great. But this can also be refined this using the Logical NOT operator for shorter and easier readability 😀

`
toDoItemsArray[itmId].completed = !completed;
`